storage/tests/WindowAggregationOperationState_unittest.cpp

```diff
@@ -0,0 +1,91 @@
+/**
+ *   Copyright 2011-2015 Quickstep Technologies LLC.
+ *   Copyright 2015-2016 Pivotal Software, Inc.
+ *   Copyright 2016, Quickstep Research Group, Computer Sciences Department,
+ *     University of Wisconsin—Madison.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ **/
+
+#include <cstddef>
+#include <memory>
+
+#include "catalog/CatalogDatabase.hpp"
+#include "catalog/CatalogRelation.hpp"
+#include "catalog/CatalogTypedefs.hpp"
+#include "expressions/aggregation/AggregateFunction.pb.h"
+#include "storage/WindowAggregationOperationState.hpp"
+#include "storage/WindowAggregationOperationState.pb.h"
+
+#include "gtest/gtest.h"
+
+using std::unique_ptr;
+
+namespace quickstep {
+
+class WindowAggregationOperationStateProtoTest : public ::testing::Test {
+ protected:
+  static const relation_id kInvalidTableId = 100;
+  static const std::int64_t kInvalidNum = -10;
+  static const std::int64_t kValidNum = 10;
```

Comment:
Optionally, like [here](https://github.com/apache/incubator-quickstep/blob/master/relational_operators/tests/SortRunGenerationOperator_unittest.cpp#L80) we could move these three values in an anonymous namespace:

```
namespace {
constexpr relation_id kInvalidTableId = 100;
constexpr std::int64_t kInvalidNum = -10;
constexpr std::int64_t kValidNum = 10;
}  // namespace
```